chains/Schain.cpp

```diff
@@ -1245,6 +1332,16 @@ ptr< BlockProposal > Schain::createDefaultEmptyBlockProposal( block_id _blockId
 
 void Schain::finalizeDecidedAndSignedBlock( block_id _blockId, schain_index _proposerIndex,
     const ptr< ThresholdSignature >& _thresholdSig ) {
+#ifdef BITE
+    std::thread( [=]() {
+        finalizeDecidedAndSignedBlockInThread( _blockId, _proposerIndex, _thresholdSig );
```

Comment:
why can this thread be safely detached? can proposal downloading be stuck?

Reply:
This is work in progress for the prototype

It will be improved